src/test/java/org/fundacionjala/trello/pages/card/CardPage.java

```diff
@@ -10,17 +10,30 @@
 import java.util.List;
 
 public final class CardPage extends PageObject {
-    private static final String CARD_NAMES_LIST = "//textarea[contains(text(), '%s')]/parent::div/"
+    private static final String GETS_ALL_CARDS_IN_A_LIST = "//textarea[contains(text(), '%s')]/parent::div/"
             + "parent::div//span[@class='list-card-title js-card-name']";
     private static final String ADD_NEW_CARD_BUTTON = "span.js-add-a-card";
     private static final String ALL_CARDS_LIST = "a.list-card";
+    private static final String CARD_TITLE = "//*[@id=\"chrome-container\"]/div[3]/div/div/div/div[3]/div[1]/h2";
+    private static final String CARD_DESCRIPTION = "//p/parent::div/div[@class='current markeddown hide-on-edit"
+            + " js-desc js-show-with-desc']";
+    private static final String CARD_TITLE_TEXT_AREA = "mod-card-back-title js-card-detail-title-input";
```

Comment:
I think that this locator is missing the "." that represent a class attribute in css.
I would write it like this: `.mod-card-back-title.js-card-detail-title-input`

Reply:
ok, ty